KMNavigationBarTransition/UIViewController+KMNavigationBarTransition.m

```diff
@@ -59,12 +60,14 @@ - (void)km_viewWillLayoutSubviews {
     UIViewController *toViewController = [tc viewControllerForKey:UITransitionContextToViewControllerKey];
     
     if ([self isEqual:self.navigationController.viewControllers.lastObject] && [toViewController isEqual:self]) {
-        [tc containerView].backgroundColor = [UIColor whiteColor];
```

Comment:
if this way is still not feasible, could you give me a demo to help me to find out the problem?